README.md

```diff
@@ -1,7 +1,11 @@
 # Permissioning Smart Contracts
 
-**Note:** The contracts herein have been third party audited. [Please
-contact us](https://consensys.net/quorum/contact-us) if you would like more information, or if you are considering using this Dapp in a production environment.
+## Production use
+[Please
+contact us](https://consensys.net/quorum/contact-us) if you are considering using this Dapp in a production environment.
```

Comment:
Did you mean to add this line break here?

Reply:
nope. fixed